src/Components/Modal/ModalError.tsx

```diff
@@ -0,0 +1,68 @@
+import { color, Sans } from "@artsy/palette"
+import { ModalWrapper } from "Components/Modal/ModalWrapper"
+import React from "react"
+import styled from "styled-components"
+
+interface ModalErrorProps extends React.HTMLProps<HTMLDivElement> {
+  show?: boolean
+  headerText?: string
+  detailText?: string
+  closeText?: string
+  onClose?: () => void
+}
+
+export class ModalError extends React.Component<ModalErrorProps> {
```

Comment:
I wonder if this should be called `ErrorModal`? `ModalError` implies that it's a class that extends `Error` (the error class).

Reply:
I was trying to keep everything under `Modal` namespaced that way (`Modal...`). But `ErrorModal` does sound better. I'll update and we can always change.